pkg/api/serve.go

```diff
@@ -33,7 +32,24 @@ const (
 	extensionValidationExcludeBody = "x-validation-exclude-body"
 )
 
-func Serve(cfg config.Config, catalog *catalog.Catalog, middlewareAuthenticator auth.Authenticator, authService auth.Service, authenticationService authentication.Service, blockAdapter block.Adapter, metadataManager auth.MetadataManager, migrator Migrator, collector stats.Collector, cloudMetadataProvider cloud.MetadataProvider, actions actionsHandler, auditChecker AuditChecker, logger logging.Logger, gatewayDomains []string, snippets []params.CodeSnippet, pathProvider upload.PathProvider, usageReporter stats.UsageReporterOperations) http.Handler {
+func Serve(
```

Comment:
It was too damn long for a single line.
Removed `cloudMetadataProvider`, and renamed `middlewareAuthenticator` to `authenticator` (not related, but simplifies).

Reply:
Can you please provide the style template you are using, if we change to something like this I need to break it down manually on every refactor.

Reply:
Adding this to the list of topics for the Code-Style effort - 
So this will be done systematically.